crates/emitter/src/emitter.rs

```diff
@@ -1259,6 +1266,21 @@ impl VerylWalker for Emitter {
         self.semicolon(&arg.semicolon);
     }
 
+    /// Semantic action for non-terminal 'LocalparamDeclaration'
+    fn type_def_declaration(&mut self, arg: &TypeDefDeclaration) {
+        self.str("typedef");
```

Comment:
The following code is better. This is because the original token holds comments and spacing around the token.

`self.token(&arg.r#type.type_token.replace("typedef"));`

Reply:
Fixed.